docroot/.htaccess

```diff
@@ -115,17 +115,8 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
-  # Return Status 403 (Forbidden) on non-ODE server access in the
```

Comment:
It seems like removing this would allow people to bypass dev,test,int urls such as www-dev-ac.cancer.gov.

Reply:
Yeah, I removed too much.  That does indeed allow the lower tiers to be accessed by their native names.  (But it does force HTTPS! 😁)